### DISCUSSION TITLE
Can we change the logo?

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1344] [Q&A]
It's not an exe 
![微信截图_20230806003350](https://github.com/zauberzeug/nicegui/assets/25096045/d353b9cd-15f8-4f07-8967-19ee33c4164b)

**Comment by natankeddem** [comment]
I am using an emoji as mine.
`ui.run(title="hush", favicon="🙉", reload=False, dark=True)`

**Reply by v479038280** [reply]
When native=True ,the logo didn't work
